components/sup/src/util/pkg.rs

```diff
@@ -37,33 +36,41 @@ pub fn install<T>(ui: &mut T,
-                                             // client is written. Anything that calls the current
-                                             // function, though, should always have a channel. We
-                                             // should
-                                             // push this "Option-ness" as far down the stack as we
-                                             // can,
-                                             // with the ultimate goal of eliminating it
-                                             // altogether.
-                                             channel,
-                                             install_source,
-                                             PRODUCT,
-                                             VERSION,
-                                             fs_root_path,
-                                             &fs::cache_artifact_path(None::<String>),
-                                             auth_token.as_ref().map(String::as_str),
-                                             // TODO fn: pass through and enable offline install
-                                             // mode
-                                             &InstallMode::default(),
-                                             // TODO (CM): pass through and enable ignore-local
-                                             // mode
-                                             &LocalPackageUsage::default(),
-                                             // Install hooks are run when the supervisor loads the
-                                             // package
-                                             // in add_service so it is repetitive to run them here
-                                             InstallHookMode::Ignore).map_err(SupError::from)
+                                                     url,
+                                                     // We currently need this to be an option due
+                                                     // to how
+                                                     // the depot
+                                                     // client is written. Anything that calls the
+                                                     // current
+                                                     // function, though, should always have a
+                                                     // channel. We
+                                                     // should
+                                                     // push this "Option-ness" as far down the
+                                                     // stack as we
+                                                     // can,
+                                                     // with the ultimate goal of eliminating it
+                                                     // altogether.
+                                                     channel,
```

Comment:
Maybe re-wrap these comments so they're less awkwardly line-broken?

Reply:
oof, yeah, that's bad... will fix.